src/sql/AtlasSQLite.cpp

```diff
@@ -0,0 +1,90 @@
+#include "AtlasSQLite.h"
+
+namespace atlas {
+
+AtlasSQLite::AtlasSQLite()
+{
+	bConnected = false;
+}
+
+AtlasSQLite::~AtlasSQLite()
+{
+	closeConnection();
+	fprintf(stdout, "Connection closed\n");
+}
+
+void AtlasSQLite::closeConnection()
+{
+	if(pCon)
+		sqlite3_close(pCon);
+}
+
+bool AtlasSQLite::isConnected()
+{
+    return bConnected;
+}
+
+bool AtlasSQLite::openConnection(std::string databaseName, std::string databaseDir)
+{
+	SQLiteDatabaseName = databaseName;
+	SQLiteDBPath	   = databaseDir;
+	
+	bConnected = true;
+
+	std::string db=SQLiteDatabaseName;
+	std::string dir=SQLiteDBPath;
+	std::string path=dir.append(db);
+
+	int rc = sqlite3_open(path.c_str(), &pCon);
+	
+	if(rc)
+	{
+        fprintf(stderr, "Can't open database: %s\n", sqlite3_errmsg(pCon));
+	    bConnected = false;
+	}
+
+    /* Create table 'IdentityPSK' if not exists */
+
+    sql = "CREATE TABLE IF NOT EXISTS IdentityPSK("  \
```

Comment:
Please put this query in a constant string in an anonymous namespace in this file